student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super()
+    @classroom = classroom
+  end
```

Comment:
Nice job implementing the student class to inherit from the person class😉. But just as required, the student class should extend the parent's (Person) constructor (name, age, and parent _permission)   by adding classroom to it😉. You can see the suggestions below and make the changes as required👍🏼.
<img width="902" alt="image" src="https://github.com/badger-99/school_library_records/assets/64643995/fe8c4a2b-511a-40a7-bb41-23aee1b7f5eb">


```suggestion
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end

```